### PR TITLE
Trapito

### PR DESCRIPTION
**Pull request by delmoralcristian**

### CODE REVIEW

File: src/main/resources/static/index.html [hunk 1]
```diff
@@ -15,11 +15,14 @@
 <script src="modules/auth/AuthService.js"></script>
 <script src="modules/auth/AuthCtrl.js"></script>
 
+
 <!-- Bootstrap core JavaScript
 ================================================== -->
 <!-- Placed at the end of the document so the pages load faster -->
 <script src="js/vendor/jquery/jquery.min.js"></script>
 <script src="js/vendor/bootstrap/bootstrap.min.js"></script>
```
Comment (ngonzalezparra) [inline src/main/resources/static/index.html]: Bootstrap components will not work properly in angular. Check this library https://angular-ui.github.io/bootstrap/

Comment (training-committee) [inline src/main/resources/static/index.html]: Nico, this is something that I have added with the design on the last commit pushed to master. I will remove. We can talk about it.

File: src/main/groovy/com/globant/onboarding/controllers/OnboardingCtrl.groovy [hunk 2]
```diff
@@ -40,4 +41,14 @@ public class OnboardingCtrl {
     def addCourse(Course course) {
         log.debug("Adding course...")
     }
+
```
Comment (fmillone) [inline src/main/groovy/com/globant/onboarding/controllers/OnboardingCtrl.groovy]: this should be in a separated controller

File: src/main/resources/static/modules/home/HomeCtrl.js [hunk 3]
```diff
@@ -1,6 +1,7 @@
 
 angular
     .module('app')
-    .controller('HomeCtrl', ['$scope', function($scope){
+    .controller('HomeCtrl', ['$scope', '$state', function($scope, $state){
 
-    }]);
\ No newline at end of file
+
```
Comment (fmillone) [inline src/main/resources/static/modules/home/HomeCtrl.js]: revert this changes

File: src/main/resources/static/modules/trainer/Video.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
+/**
+ * Created by cristian.delmoral on 03/05/2016.
+ */
+angular
+    .module('app')
+    .controller('Video', ['$scope', function ($scope) {
```
Comment (fmillone) [inline src/main/resources/static/modules/trainer/Video.js]: the name of this controller should be VideoCtrl

File: src/main/resources/static/modules/trainer/trainerView.tpl.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
+<section role="main" tabindex="-1">
+    <div class="login-panel content-body" ng-controller="Video">
+        <div class="login-inner">
+            <section class="login-content"><h2>Upload video</h2>
+                <form>
+                    <!--<img src="https://help.switch.ch/export/sites/help/cast/.galleries/images/VideoUpload.jpg" width="141" height="32">-->
```
Comment (fmillone) [inline src/main/resources/static/modules/trainer/trainerView.tpl.html]: remove this comment

File: src/main/resources/static/modules/trainer/Video.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+        $scope.IsMatch = false;
```
Comment (fmillone) [inline src/main/resources/static/modules/trainer/Video.js]: this statement should be in a init function

File: src/main/groovy/com/globant/onboarding/controllers/OnboardingCtrl.groovy (hunk 2 above)
Comment (delmoralcristian) [inline src/main/groovy/com/globant/onboarding/controllers/OnboardingCtrl.groovy]: done
